src/Components/test/testassets/TestContentPackage/TestContentPackage.csproj

```diff
@@ -7,6 +7,10 @@
     <StaticWebAssetBasePath>_content/TestContentPackage</StaticWebAssetBasePath>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <EnableTypeScriptNuGetTarget>true</EnableTypeScriptNuGetTarget>
```

Comment:
I mentioned this in another comment, but want to extend the question to why this appears in a bunch of places.

Reply:
There's an issue filed about this.

Reply:
https://github.com/dotnet/aspnetcore/issues/18980